article/app/controllers/ArticleController.scala

```diff
@@ -104,26 +110,35 @@ object ArticleController extends Controller with RendersItemResponse with Loggin
       renderFormat(htmlResponse, jsonResponse, article, Switches.all)
   }
 
-  def renderArticle(path: String, lastUpdate: Option[String], rendered: Option[Boolean]) = {
+  def renderArticle(path: String, lastUpdate: Option[String], rendered: Option[Boolean], page: Option[String]) = {
```

Comment:
I think Play will just do that, but if not, I'd definitely try and convert it here, rather than further down the stack.